src/pages/MainPage.jsx

```diff
@@ -0,0 +1,96 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import CardMedia from '@mui/material/CardMedia';
+import Grid from '@mui/material/Grid';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+const cards = [1, 2, 3];
+
+const theme = createTheme();
+
+export default function MainPage() {
+  return (
+    <ThemeProvider theme={theme}>
+      <Grid
+        container
+        alignItems="center"
+        spacing={0}
+        justifyContent="center"
+        direction="column"
+      >
+        <main>
+          <Box
+            sx={{
+              bgcolor: 'background.paper',
+              pt: 8,
+              pb: 6,
+            }}
+          >
+            <Container maxWidth="sm">
+              <Typography
+                component="h2"
+                variant="h2"
+                align="center"
+                color="text.primary"
+                sx={{ paddingTop: '50px' }}
+                gutterBottom
+              >
+                Allen Institute for Neural Dynamics
+              </Typography>
+              <Typography
+                variant="h5"
+                align="center"
+                color="text.secondary"
+                paragraph
+              >
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+                enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
+                in reprehenderit in voluptate velit esse cillum dolore eu fugiat
+                nulla pariatur. Excepteur sint occaecat cupidatat non proident,
+                sunt in culpa qui officia deserunt mollit anim id est laborum.
```

Comment:
Assume that this will be public facing. So don't use "lorem ipsum" as placeholder text.